emcc.py

```diff
@@ -1749,7 +1749,7 @@ def get_final():
 
     with ToolchainProfiler.profile_block('source transforms'):
       # Embed and preload files
-      if len(options.preload_files) + len(options.embed_files) > 0:
+      if len(options.preload_files) + len(options.embed_files):
```

Comment:
This one looks really weird to me. I feel like I'd prefer it as `len(a) + len(b) > 0`, or `len(a) or len(b)`, or `a or b`